build/build.xml

```diff
@@ -386,6 +394,7 @@
 		<build-addon name="cmss" />
 		<build-addon name="codedx"/>
 		<build-addon name="communityScripts"/>
+		<build-addon name="cspscanner"/>
```

Comment:
Ok so this fixes something missed in https://github.com/zaproxy/zap-extensions/pull/968?

I'm fine with it being included here, just want to make sure it's doc'd somehow..